tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_catchBlockUniqueParamsBindingPattern.js

```diff
@@ -0,0 +1,20 @@
+// ==ORIGINAL==
+
+function /*[#|*/f/*|]*/() {
+	return Promise.resolve().then(() => ({ x: 3 })).catch(() => ({ x: "a" })).then(({ x }) => !!x);
+}
+
+// ==ASYNC FUNCTION::Convert to async function==
+
+async function f() {
+	let result;
```

Comment:
Seems like this case is causing some odd formatting (i.e. indentation on the `let`)?
I don't think it's that big a deal, but users might point it out

Reply:
Weird, that's a tab whereas the other lines have spaces... maybe it's an artifact of git or VS Code?

_[investigates]_

There are inconsistent tabs vs. spaces in the test case. Updated to avoid confusion. Good catch!